cluster/gce/manifests/network-proxy.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: network-proxy
+  namespace: kube-system
+  annotations:
+    scheduler.alpha.kubernetes.io/critical-pod: ''
+    seccomp.security.alpha.kubernetes.io/pod: 'docker/default'
+  component: network-proxy
+spec:
+  hostNetwork: true
+  containers:
+  - name: network-proxy-container
+    image: gcr.io/google-containers/proxy-server:v0.0.3
+    resources:
+      requests:
+        cpu: {{ cpulimit }}
```

Comment:
Since this is hard coded anyway, why not just write it here directly?

Reply:
Fixed.